packages/@okta/vuepress-site/docs/concepts/interaction-code/index.md

```diff
@@ -0,0 +1,100 @@
+
+## The Interaction Code flow
+
+The Interaction Code flow is similar to the [OAuth 2.0 Authorization Code flow with PKCE](/docs/concepts/oauth-openid/#authorization-code-flow-with-pkce). All clients are required to pass along a client ID, as well as a Proof Key for Code Exchange (PKCE), to keep the flow secure. Confidential clients such as web apps must also pass a client secret in their authorization request. The user can start the authorization request with minimal information, relying on the client to facilitate the interactions with the Okta org to progressively authenticate the user. The series of interactions, which could include multifactor authentication steps, is secured using the `interaction_handle`. After successfully completing the remedial interactions, the client receives an `interaction_code` that they can then redeem for tokens from an Okta Authorization Server with the [`/token`](/docs/reference/api/oidc/#token) endpoint.
+
+The Interaction Code flow is supported by Okta's two authorization server types: the [Org Authorization Server](/docs/concepts/auth-servers/#org-authorization-server) and [Custom Authorization Server](/docs/concepts/auth-servers/#custom-authorization-server). These Okta Authorization Servers must have the Interaction Code grant type configured in their Access Policy Rule to accept Interaction Code requests.
+
+The following table describes the parameters introduced for the Interaction Code grant type flow:
+
+| Interaction Code grant parameter           | Description   |
+| --------------------------------           | -----------   |
+| `interaction_code` |  The `interaction_code` is a one-time use, opaque code that the client can exchange for tokens using the Interaction Code grant type. This code enables a client to redeem a completed Identity Engine interaction for tokens without needing access to an authorization server’s session. |
+| `interaction_handle` | The `interaction_handle` is an opaque, immutable value that is provided by an Okta Authorization Server. The client can use the `interaction_handle` to interact with the Identity Engine component of the Okta org directly. The client is responsible for saving the `interaction_handle` and using it to hold the state of the transaction during remediation. Any public/confidential client that is configured to use the Interaction Code grant type can obtain an `interaction_handle`. If all the remediation steps are successfully performed, an `interaction_code` is returned as part of the success response.            |
```

Comment:
`interaction_handle` is provided by `Identity Engine` though not sure if that may be a little confusing if we will use `Identity Engine` instead of `Okta Authorization Server`.

Reply:
also, perhaps having `interaction_handle` be the first in the table may make sense since that is the first thing customers are going to get.